Dockerfile

```diff
@@ -42,9 +43,15 @@ WORKDIR /srv/selection-tool
 COPY /requirements.txt /srv/selection-tool/requirements.txt
 RUN pip install -r requirements.txt
 
-#install the schematron-nokogiri gem
-RUN gem install schematron-nokogiri
+#install the schematron-nokogiri gem from BuildingSync org
+WORKDIR /srv
+RUN git clone https://github.com/BuildingSync/schematron
```

Comment:
yay! looks like we had to do some tweaks?

Reply:
yep!